ui/src/submissions/conferences/containers/ExistingConferencesAlertContainer.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useMemo } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { Button } from 'antd';
+import FieldInfoAlert from '../../common/components/FieldInfoAlert';
+import pluralizeUnlessSingle from '../../../common/utils';
+import ExistingConferencesDrawer from '../components/ExistingConferencesDrawer';
+import { EXISTING_CONFERENCES_NS } from '../../../reducers/search';
+import {
+  RANGE_AGGREGATION_SELECTION_SEPARATOR,
+  START_DATE_ALL,
+} from '../../../common/constants';
+import { searchQueryUpdate } from '../../../actions/search';
+
+export function ExistingConferencesAlert({
+  dispatch,
+  dates,
+  numberOfConferences,
+}) {
+  const [openingDate, closingDate] = dates;
+  const [showDrawer, setShowDrawer] = useState(false);
+
+  useMemo(
```

Comment:
`useMemo`, memoizing "expensive" operations to improve performance, you should use `useEffect` for side effects like dispatching actions.